translations/es.po

```diff
@@ -710,40 +710,41 @@ msgid ""
 "  ( -i | --ignore-case )           Ignore case differences\n"
 "  ( -w | --ignore-all-space )      Ignore white space differences"
 msgstr ""
-"Usage:\n"
+"Uso:\n"
 "    diffuse [ [OPTION...] [FILE...] ]...\n"
 "    diffuse ( -h | -? | --help | -v | --version )\n"
 "\n"
-"Diffuse is a graphical tool for merging and comparing text files.  Diffuse is\n"
-"able to compare an arbitrary number of files side-by-side and gives users the\n"
-"ability to manually adjust line matching and directly edit files.  Diffuse can\n"
-"also retrieve revisions of files from Bazaar, CVS, Darcs, Git, Mercurial,\n"
-"Monotone, RCS, Subversion, and SVK repositories for comparison and merging.\n"
+"Diffuse es una herramienta gráfica para la mezcla y comparación de ficheros de\n"
+"texto. Diffuse puede comparar un número arbitrario de ficheros lado a lado y\n"
+"da a los usuarior la capacidad de realizar un ajuste de lineas manual y de\n"
+"editar los ficheros directamente. Diffuse puede también recuperar versiones\n"
+"de ficheros a partir de repositorios de Bazaar, CVS, Darcs, Git, Mercurial,\n"
+"Monotone, RCS, Subversion y SVK para su comparación y mezcla.\n"
 "\n"
-"Help Options:\n"
-"  ( -h | -? | --help )             Display this usage information\n"
-"  ( -v | --version )               Display version and copyright information\n"
+"Opciones de ayuda:\n"
+"  ( -h | -? | --help )           Muestra esta información de uso\n"
+"  ( -v | --version )             Muestra información de versionado y copyright\n"
 "\n"
-"Configuration Options:\n"
-"  --no-rcfile                      Do not read any resource files\n"
-"  --rcfile <file>                  Specify explicit resource file\n"
+"Opciones de configuración:\n"
+"  --no-rcfile                    No leer ficheros de recursos\n"
+"  --rcfile <file>                Especificar fichero de recursos explícitamente\n"
 "\n"
-"General Options:\n"
-"  ( -c | --commit ) <rev>          File revisions <rev-1> and <rev>\n"
-"  ( -D | --close-if-same )         Close all tabs with no differences\n"
-"  ( -e | --encoding ) <codec>      Use <codec> to read and write files\n"
-"  ( -m | --modified )              Create a new tab for each modified file\n"
-"  ( -r | --revision ) <rev>        File revision <rev>\n"
-"  ( -s | --separate )              Create a new tab for each file\n"
-"  ( -t | --tab )                   Start a new tab\n"
-"  --null-file                      Create a blank file comparison pane\n"
+"Opciones generales:\n"
+"  ( -c | --commit ) <rev>        Revisiones de fich. <rev-1> y <rev>\n"
+"  ( -D | --close-if-same )       Cerrar pestañas sin diferencias\n"
+"  ( -e | --encoding ) <cod>      Usa codificación <cod> al leer y escribir\n"
```

Comment:
I don't speak Spanish, but it seems like "cod" doesn't exist. Although Google Translate gives me "códec", I think keeping "`<codec>`" would be fine, since it is more part of the API than the description (as for OPTION and FILE above).

Reply:
Resolved by #43